app/controllers/SubmitJobController.scala

```diff
@@ -0,0 +1,27 @@
+package controllers
+
+import javax.inject._
+import play.api._
+import play.api.i18n.I18nSupport
+import play.api.mvc._
+
+import forms.JobForm
+
+class SubmitJobController @Inject()(
+  cc: ControllerComponents
+) extends AbstractController(cc) with I18nSupport {
+
+  def view() = Action { implicit request: Request[AnyContent] =>
```

Comment:
I think usually for parameterless methods without side-effects would not include parenthesis.

Reply:
my bad I didn't do this on the `HomeController`